CreateMessage/index.ts

```diff
@@ -30,26 +29,26 @@ secureExpressApp(app);
 // Set up CORS (free access to the API from browser clients)
 app.use(cors());
 
+// Setup DocumentDB
+const cosmosDbUri = getRequiredStringEnv("COSMOSDB_URI");
 const cosmosDbName = getRequiredStringEnv("COSMOSDB_NAME");
-const messageContainerName = getRequiredStringEnv("MESSAGE_CONTAINER_NAME");
+const cosmosDbKey = getRequiredStringEnv("COSMOSDB_KEY");
 
-const documentDbDatabaseUrl = documentDbUtils.getDatabaseUri(cosmosDbName);
-const messagesCollectionUrl = documentDbUtils.getCollectionUri(
-  documentDbDatabaseUrl,
-  MESSAGE_COLLECTION_NAME
-);
-const servicesCollectionUrl = documentDbUtils.getCollectionUri(
-  documentDbDatabaseUrl,
-  SERVICE_COLLECTION_NAME
-);
+const cosmosdbClient = new CosmosClient({
+  endpoint: cosmosDbUri,
+  key: cosmosDbKey
+});
+
```

Comment:
Maybe, we can extract and refactor this code into a util file

Reply:
yes, there should be only one singleton client shared between functions